README.md

```diff
@@ -22,23 +22,45 @@ Read on to learn about the details!
 
 Install the app into your environment:
 
-> ```pip install django_components```
+```> pip install django_components```
 
 Then add the app into INSTALLED APPS in settings.py
 
-```
+```python
 INSTALLED_APPS = [
     ...,
     "django_components",
     ...
 ]
 ```
 
+Modify `TEMPLATES` section of settings.py as follows:
+- Remove `'APP_DIRS': True,`
```

Comment:
Does this really have to be removed? As I understand it it just adds more directories to look into for templates. I can imagine people needing this setting for other reasons.

Reply:
Yep, otherwise Django will complain. `APP_DIRS = True` enables `django.template.loaders.app_directories.Loader` and as we have to add our own loader we must put it in `loaders` explicitly, `APP_DIRS` becomes redundant. Refer to this section https://docs.djangoproject.com/en/3.2/ref/templates/api/#django.template.loaders.cached.Loader and above for clarification.